Set3/17/findDouble.cc

```diff
@@ -0,0 +1,16 @@
+#include "sum.ih"
+
+bool findDouble(size_t const argc, char const *argv[])
+{
+    bool foundDouble = false; 
+    for (size_t argIdx = 1; argIdx != argc; ++argIdx) 
+    {
+        string const argument = argv[argIdx];                                 
+        if (argument.find(".") != string::npos)                               
+        { 
+            foundDouble = true; 
+            break;                              // One double is enough
+        }       
+    }
+    return foundDouble;
```

Comment:
```suggestion
    for (size_t argIdx = 1; argIdx != argc; ++argIdx) 
    {
        string const argument = argv[argIdx];                                 
        if (argument.find(".") != string::npos)                               
            return true; 				// One double is enough
    }
    return false;
```

The `foundDouble` variable is superfluous. Instead, we can directly return the corresponding bool.